ddtrace/opentelemetry/span.go

```diff
@@ -171,3 +203,115 @@ func toSpecialAttributes(k string, v attribute.Value) (string, interface{}) {
 		return k, v.AsInterface()
 	}
 }
+
+const (
+	httpServerKey    = "http.server.request"
+	httpClientKey    = "http.client.request"
```

Comment:
Why make these consts and not the rest? These should be fine to inline like the others. Either way, we should just be consistent.